app/models/user_export.rb

```diff
@@ -3,10 +3,8 @@ class UserExport < ActiveRecord::Base
 
   def self.remove_old_exports
     UserExport.where('created_at < ?', 2.days.ago).find_each do |user_export|
-      file_name = "#{user_export.file_name}-#{user_export.id}.csv.gz"
-      file_path = "#{UserExport.base_directory}/#{file_name}"
```

Comment:
I think we can nuke `UserExport.base_directory` from our code base as well. We do need to think about how to clean up that folder which is left around.

Reply:
We still need `UserExport.base_directory` for creating temporary files. Relevant code [here](UserExport.base_directory).

Reply:
Hmm I feel like tmp files should go into a tmp directory so that we don't affect the uploads directory.

Reply:
Okay, I'll get that done in separate commit.